lib/suggestions.js

```diff
@@ -9,8 +9,23 @@ import { Range } from 'atom'
 
 function resolveModule(textEditor, module) {
     const basedir = path.dirname(textEditor.getPath())
+
+    const moduleDirectory = [ 'node_modules' ]
+    const webpackConfigs = atom.config.get('js-hyperclick.webpackConfigs')
+    webpackConfigs.forEach(configFileName => {
+        try {
+            const config = require(path.join(atom.project.getPaths()[0], configFileName))
```

Comment:
Do you mean use `fs.accessSync`? I thought it throws an error if there is no file (or any specified mode has not met).